src/app/pages/dashboard/components/practice-end-to-end-card/practice-end-to-end-card.html

```diff
@@ -1,12 +1,12 @@
 <button clear (click)="navigateToFakeJournal()">
-  <ion-card class="practice-mode-card">
+  <ion-card class="practice-mode-card" id="practice-mode-card">
```

Comment:
can you make this distinct from the id for practice-test-report-card. it would cause issues as they're both on the same parent page

Reply:
Changed to reflect PR commit